packages/core/database/src/entity-manager/regular-relations.ts

```diff
@@ -23,6 +23,36 @@ declare module 'knex' {
   }
 }
 
+//  TODO: This is a short term solution, to not steal relations from the same document.
+const getDocumentSiblingIdsQuery = (con: Knex, tableName: string, id: ID) => {
+  // Find if the model is a content type or something else (e.g component)
+  // to only get the documentId if it's a content type
+  const models: Model[] = Array.from(strapi.db.metadata.values());
```

Comment:
probably best to add the model in the deleteRelation & othe relational methods signature eventually (let's refacto later)